types/src/hard_fork.rs

```diff
@@ -47,58 +69,87 @@ pub enum HardFork {
     V13,
     V14,
     V15,
-    // remember to update from_vote!
     V16,
 }
 
 impl HardFork {
+    /// The current [`HardFork`].
+    ///
+    /// ```rust
+    /// # use cuprate_types::HardFork;
+    /// assert_eq!(HardFork::CURRENT, HardFork::V16);
+    /// ```
+    pub const CURRENT: Self = Self::VARIANTS[Self::COUNT - 1];
```

Comment:
Bit of nit but I think this should be called  `LATEST`  as the last HF may have not been activated, therefore it's not always the current HF.